python/ray/_private/worker.py

```diff
@@ -1203,6 +1203,7 @@ def init(
     namespace: Optional[str] = None,
     runtime_env: Optional[Union[Dict[str, Any], "RuntimeEnv"]] = None,  # noqa: F821
     storage: Optional[str] = None,
+    labels: Optional[Dict[str, str]] = None,
```

Comment:
Since these are kwargs and the ordering doesn't matter, let's put this next to `resources`, that will logically be the best place to put it :)